### ISSUE TITLE
property-no-unknown does not recognize new properties in Chrome 85+

### ISSUE THREAD

**Issue by victor-homyakov** [stylelint/stylelint#4966]
> Clearly describe the bug

Properties recently added to Chrome 85+, e.g. `content-visibility`, are not recognized by the rule

> Which rule, if any, is the bug related to?

`property-no-unknown`

> What code is needed to reproduce the bug?

```css
.lazy {
    content-visibility: auto;
    contain-intrinsic-size: 400px;
}
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
    "rules": {
        "property-no-unknown": true
    }
}
```

> Which version of stylelint are you using?

`13.7.2`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

1. stylelint is installed as dev dependency
2. `.stylelintrc` is stored in the project's root folder
3. CLI with `npx stylelint path/to/my.css`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:

```shell
path/to/my.css
 2:5  ✖  Unexpected unknown property "content-visibility"       property-no-unknown
 3:5  ✖  Unexpected unknown property "contain-intrinsic-size"   property-no-unknown
```

How to fix: bump known-css-properties to latest (0.20.0) - it already contains these new properties
